### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by Oxunjon1029**

### CODE REVIEW

File: routes/authentication.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+router.post('/',(req,res)=>{
+  res.send("Authenticated succesfully");
```
Comment (AudriusBaltusis) [inline routes/authentication.js:5]: Looks like something is missing here

File: routes/categories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+router.post('/add', (req, res) => {
+  res.send("Category created succesfully");
```
Comment (AudriusBaltusis) [inline routes/categories.js:5]: Looks like something is missing here

File: routes/account.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    res.status(200).json(account);
+  } catch (err) {
+    res.status(400).json(err.message);
+  }
+})
+
+
+router.get('/', async (req, res) => {
+  const account = await Account.find({});
```
Comment (AudriusBaltusis) [inline routes/account.js:21]: this logic should be in controller. route should look like `router.get('/', Account.find)`

File: routes/account.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  res.json(account)
+})
+
+
+router.put('/edit', async (req, res) => {
+  try {
```
Comment (AudriusBaltusis) [inline routes/account.js:27]: such routes should be protected by `auth` guard

**Review by AudriusBaltusis — CHANGES_REQUESTED** [review]

**Comment by Oxunjon1029** [comment]
done

**Review by AudriusBaltusis — APPROVED** [review]

**Pull request merged**